addons/point_of_sale/static/tests/tours/ticket_screen_tour.js

```diff
@@ -78,7 +78,10 @@ registry.category("web_tour.tours").add("TicketScreenTour", {
             TicketScreen.clickControlButton("Print Receipt"),
             TicketScreen.receiptTotalIs("8.00"),
             ReceiptScreen.clickBack(),
-            TicketScreen.clickBackToMainTicketScreen(),
+            {
+                ...{ ...ProductScreen.back(), mobile: true },
+                mobile: true,
+            },
```

Comment:
Why do you need to put mobile two times ?

Reply:
indeed, i do not. It's fixed now